sdk/python/v1beta1/kubeflow/katib/types/__init__.py

```diff
@@ -0,0 +1,12 @@
+# coding: utf-8
+
+# flake8: noqa
+
+
+from __future__ import absolute_import
+
+# Import types into type package.
+from kubeflow.katib.types.trainer_resources import TrainerResources
+
+# Import Kubernetes models.
+from kubernetes.client import *
```

Comment:
Do you need this file ?

If users should be able to run this: `from kubeflow.katib import TrainerResources`, just update this script: https://github.com/kubeflow/katib/blob/master/hack/gen-python-sdk/post_gen.py#L43 with this line:
```
 lines.append("from kubeflow.katib.types.trainer_resources import TrainerResources\n")
 ```

Reply:
Thank you for your suggestions! I've updated the code accordingly.